lib/resque/data_store.rb

```diff
@@ -97,7 +97,7 @@ def server_time
     end
 
     def redis_time_available?
-      @redis.time
+      @redis.time unless Resque.inline
```

Comment:
Can you explain the consequences of this change? Does this mean that the server-time will never be used (even if the server supports it) unless you have inlining disabled?